homeassistant/components/sonos/media_player.py

```diff
@@ -427,6 +430,8 @@ def device_info(self):
             "identifiers": {(SONOS_DOMAIN, self._unique_id)},
             "name": self._name,
             "model": self._model.replace("Sonos ", ""),
+            "sw_version": self._sw_version,
+            "connections": {(dr.CONNECTION_NETWORK_MAC, self._mac_address)},
```

Comment:
Both identifiers and connections aren't needed. One of those is enough.

Reply:
ah so one can be overly identified interesting. For my own curiousity, what if one is serial number and other is MAC? It seems serial souod always be preferred?

Reply:
Yeah, I would also say that serial is preferred as id. Mac address has some limitations depending on the context.

Reply:
I can remove `connection` but I was under the impression it was preferred over `identifiers` based on https://github.com/home-assistant/core/pull/33165#discussion_r396204164